File: Sortings/bubble_sort.py
def bubble_sort(lst):
    for i in range(len(lst)):
        for j in range(len(lst)-1):
            if lst[j] > lst[j+1]:
                lst[j], lst[j+1] = lst[j+1], lst[j]

if __name__ == "__main__":
    lst = [9, 7, 5, 6, 2, 1, 4, 3, 0, 8]
    print("Unsorted list")
    print(lst)
    print("Sorted list")
    bubble_sort(lst)
    print(lst)

File: Sortings/bubbleSort.cpp
#include<bits/stdc++.h>
using namespace std;

void swap(int& a, int& b) {
    int aux = a;
    a = b;
    b = aux;
}

void bubbleSort(int arr[], int size) {
    for(int i=0; i<size; ++i) {
        for(int j=0; j<size-1; ++j) {
            if(arr[j] > arr[j+1]) 
                swap(arr[j], arr[j+1]);
        }
    }
}

void printArr(int arr[], int size) {
    for(int i=0; i<size; ++i)
        cout << arr[i] << " ";
    cout << "\n";
}

int main() {
    int size = 10;
    int arr[10] = {9, 7, 5, 6, 2, 1, 4, 3, 0, 8};

    cout << "Unsorted array" << endl;
    printArr(arr, size);
    cout << "Sorted array" << endl;
    bubbleSort(arr, size);
    printArr(arr, size);

    return 0;
}
